library/foundation/src/main/java/com/epmedu/animeal/foundation/theme/Theme.kt

```diff
@@ -34,12 +36,20 @@ fun AnimealTheme(
     darkTheme: Boolean = isSystemInDarkTheme(),
     content: @Composable () -> Unit
 ) {
+    val systemUiController = rememberSystemUiController()
     val colors = if (darkTheme) {
         DarkColorPalette
     } else {
         LightColorPalette
     }
 
+    LaunchedEffect(Unit) {
```

Comment:
This should be inside MaterialTheme

Reply:
Please check in dark theme

Reply:
> This should be inside MaterialTheme

Done

Reply:
> Please check in dark theme

Recorded videos with navigation bar and without it in both themes